src/OpenTelemetry.Exporter.Jaeger/Implimentation/JaegerConversionExtensions.cs

```diff
@@ -0,0 +1,142 @@
+// <copyright file="JaegerConversionExtensions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Jaeger.Implimentation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using OpenTelemetry.Common;
+    using OpenTelemetry.Trace;
+    using OpenTelemetry.Trace.Export;
+
+    public static class JaegerConversionExtensions
+    {
+        private const long MillisPerSecond = 1000L;
+        private const long NanosPerMillisecond = 1000 * 1000;
+        private const long NanosPerSecond = NanosPerMillisecond * MillisPerSecond;
+
+        public static JaegerSpan ToJaegerSpan(this SpanData span)
+        {
+            var jaegerTags = new List<JaegerTag>();
+
+            if (span?.Attributes?.AttributeMap != null)
```

Comment:
ideally hold the reference after the checks. Pattern matching works great here:
```suggestion
            if (span?.Attributes?.AttributeMap is AttributeMap attributeMap)
```
Then you can use the local variable below